setup.py

```diff
@@ -21,8 +21,8 @@
     packages=setuptools.find_packages(),
     setup_requires=['setuptools_scm'],
     install_requires=[
-        'cloud_inquisitor>=1.0.0',
-        'APScheduler>=3.3.1',
+        'cloud_inquisitor~=1.1.0',
+        'APScheduler~=3.3',
```

Comment:
Not sure if it's worth upgrading right now, but APScheduler 3.5.0 resolves a memory leak when a worker raises an exception:
https://github.com/agronholm/apscheduler/blob/master/docs/versionhistory.rst